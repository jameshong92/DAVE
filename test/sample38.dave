// test tbl convert and plus
int main() {
    rec r1 = new rec(name:"Yang", score:94); 
    rec r2 = new rec(name:"James", score:95);
    rec r3 = new rec(name:"Micheal", score:98);
    rec[] r = [r1, r2, r3];
    tbl test = new tbl(r);
    print(test);
    rec r4 = new rec(name:"Yang", score:72); 
    rec r5 = new rec(name:"James", score:90);
    rec r6 = new rec(name:"Micheal", score:88);
    rec[] r7 = [r4, r5, r6];
    tbl test2 = new tbl(r7);
    print(test2);
    test = convert(test, 1, "int");
    test = test + test2;
    print(test);
}