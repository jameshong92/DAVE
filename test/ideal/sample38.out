name	score	
Yang	94.1	
James	95.2	
Micheal	98.3	

name	score	
Yang	72	
James	90	
Micheal	88

name	score	
Yang	166	
James	185	
Micheal	186	

